ccpage178.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+# Write a class called IceCreamStand that inherits from the Restaurant class.
+# Add an attribute called flavors that stores a list of ice cream flavors.
+# Write a method that displays these flavors.
+# Create an instance of IceCreamStand, and call this method.
+class Restaurant():
+    """An attempt to model a restaurant"""
+    def __init__(self, name, cuisine):
+        """Initialize the attributes"""
+        self.name = name
+        self.cuisine = cuisine
+    
+    def describe_restaurant(self):
+        """Write two sentences to describe restaurant"""
+        print(f"The restaurant is called {self.name.title()}.")
+        print(f"{self.name.title()} serves {self.cuisine.title()}.")
+    
+    def open(self):
+        """Indicate whether or not the restaurant is open"""
+        print(f"{self.name.title()} is open.")
+    
+class IceCreamStand(Restaurant):
+    """An attempt to model a specific type of restaurant, an ice cream shop"""
+    def __init__(self, name, cuisine):
+        """Initialize the attributes"""
+        super().__init__(name, cuisine)
+        self.flavors = ['chocolate', 'vanilla', 'strawberry']
+    
+    def ice_cream_flavors(self):
+        """List the ice cream flavors offered"""
+        flavors_offered = ['chocolate', 'vanilla', 'strawberry']
```

Comment:
This list is already an attribute in the `__Init__` method, so you can just call that and not redefine it.